src/gen_dragonfruit/gd_dev_batt.cpp

```diff
@@ -8,8 +8,8 @@ int gd_dev_batt_read(void){
     int value = 555;
 
     #ifndef SEN_STUB
-    value = analogRead(_PIN_GD_BATT_)*5000.0/1023; //Function gives ADC value (0-1023), then convert
-    #endif
+    value = analogRead(_PIN_GD_BATT_)*5000.0/1023; // Function returns the battery reading as an integer in the range from 0 to 1023. 
+    #endif                                         // Divide by 1023 to offset and then convert to mV.
```

Comment:
This comment shouldn't be on the line with `#endif`. Also, what did you mean 'to offset`?